docs/faq.rst

```diff
@@ -0,0 +1,230 @@
+.. meta::
+  :description: This page lists frequently asked questions about HIP
+  :keywords: AMD, ROCm, HIP, FAQ, frequently asked questions
+
+*******************************************************************************
+Frequently asked questions
+*******************************************************************************
+
+This topic provides answers to frequently asked questions from new HIP users and
+users familiar with NVIDIA CUDA.
+
+HIP Support
+===========
+
+What NVIDIA CUDA features does HIP support?
+-------------------------------------------
+
+The :doc:`NVIDIA CUDA runtime API supported by HIP<hipify:tables/CUDA_Runtime_API_functions_supported_by_HIP>`
+and :doc:`NVIDIA CUDA driver API suupported by HIP<hipify:tables/CUDA_Driver_API_functions_supported_by_HIP>`
+pages describe which NVIDIA CUDA APIs are supported and what the equivalents are.
+The :doc:`HIP API documentation <doxygen/html/index.html>` describes each API and
+its limitations, if any, compared with the equivalent CUDA API.
+
+The kernel language features are documented in the
+:doc:`/reference/cpp_language_extensions` page.
+
+What libraries does HIP provide?
+--------------------------------
+
+HIP provides key math and AI libraries. See :doc:`rocm:reference/api-libraries`
+for the full list.
+
+What hardware does HIP support?
+-------------------------------
+
+HIP supports AMD and NVIDIA GPUs. See
+:ref:`prerequisites of the install guide<install_prerequisites>` for detailed
+information.
+
+What operating systems does HIP support?
+----------------------------------------
+
+The supported operating systems are listed in the
+:doc:`rocm:compatibility/compatibility-matrix`.
+
+CUDA and OpenCL
```

Comment:
This tittle should be removed.

Reply:
I changed it. Removing it would expand the dynamic table of contents on the right side too much, and it's also not directly related to "HIP support"

Reply:
Sure